### DISCUSSION TITLE
RxODE not available on CRAN to download

### DISCUSSION THREAD

**Discussion by sfreidy0** [nlmixrdevelopment/RxODE#378] [General]
Hi,

I was working on a project in RxODE but there was an issue where I had to uninstall and re-install R. I am trying to reinstall RxODE but it's not available on CRAN. I saw the previous comments about resubmission, Do you know when it would be available again? if not, is there a way to downloaded from somewhere else?

**Comment by mattfidler** [comment]
It is being reviewed by CRAN again, https://lockedata.github.io/cransays/articles/dashboard.html shows 4 days in review.  Its been in a few rounds of review.  In short, I am unsure how long it will take for this review.  Nor do I know if they will ask for additional changes.  During the review the status went from the folder "newbies" to "pending".  My guess "pending" means it has to run a very specialized R (USBAN/SAN for instance) and make sure there are no issues.  I make this assumption because  nothing has been sent to us on our end (I think the "pending" folder means that right now).

In the mean time you can always get from github:

```r
remotes::install_devtools("nlmixrdevelopment/RxODE")
```

**Comment by sfreidy0** [comment]
Thank you for your prompt reply.
I get the error message below when I tried your suggestion. Am I missing something? do I need to do something else before I run the command? 
" remotes::install_devtools("nlmixrdevelopment/RxODE")
Error: 'install_devtools' is not an exported object from 'namespace:remotes'"

**Comment by mattfidler** [comment]
That is my mistake it should be:

```r
remotes::install_github("nlmixrdevelopment/RxODE")
```

**Comment by sfreidy0** [comment]
Thank you for your help.
I downloaded the package but I am getting this message:
RxODE 1.0.3 using 2 threads (see ?getRxThreads)
  no cache: create with `rxCreateCache()`
What can I do?

**Comment by mattfidler** [comment]
You can either run RxODE as normal, or create a cache to speed up some processing.  If you create a cache, you can always clean the cache with `rxClean()`

**Comment by mattfidler** [comment]
This isn't really a bug and it is more of a discussion so I moved it here.

**Reply by sfreidy0** [reply]
No problem! thank you for your help.
